### PR TITLE
Uninstall fails to remove folder

### PR DESCRIPTION
**Pull request by sudhirverma**

### CODE REVIEW

File: uninstaller/uninstall-helper.ps1 [hunk 1]
```diff
@@ -55,6 +55,8 @@ $targetFolder = [System.IO.Path]::GetFullPath((Join-Path ($folder) '..'))
 
 echo 'Removing installation folder'
 
+New-Item "$folder\..\temp" -type Directory -Force | Out-Null
+robocopy "$folder\..\temp" "$targetFolder" /purge | Out-Null
 Remove-Item -path "$targetFolder" -Force -Recurse
```
Comment (dgolovin) [inline uninstaller/uninstall-helper.ps1]: This line works for me perfectly on Windows 10.  Making full copy of installation folder (imagine if everything is installed)   before deleting it does not make any sense to me.
What Windows version are you getting error on? What is the error?

Comment (sudhirverma) [reply uninstaller/uninstall-helper.ps1]: I am using Windows 7. Powershell version is 5.

error is:

error

Comment (dgolovin) [reply uninstaller/uninstall-helper.ps1]: Could you test if solution here https://stackoverflow.com/questions/38141528/cannot-remove-item-the-directory-is-not-empty would work?

Comment (sudhirverma) [reply uninstaller/uninstall-helper.ps1]: No It's not working it showing error:

error

Comment (dgolovin) [reply uninstaller/uninstall-helper.ps1]: Got the same on WIndows 10.

Comment (dgolovin) [reply uninstaller/uninstall-helper.ps1]: Fixed in this PR https://github.com/redhat-developer-tooling/developer-platform-install/pull/1084
Could you verify on Windows 7?

**Comment by dgolovin** [comment]
Closing this one in favor of https://github.com/redhat-developer-tooling/developer-platform-install/pull/1084

File: uninstaller/uninstall-helper.ps1 (hunk 1 above)
Comment (sudhirverma) [reply uninstaller/uninstall-helper.ps1]: Still not working on windows 7, it not showing any error but it's not deleting folder.

**Pull request closed without merging**